### PR TITLE
New package: hyperscan-5.1.1

### PR DESCRIPTION
**Pull request by non-Jedi**
Relatively length build time on my machine. It's possible CI won't be able to finish with this one. Built and tested on x86_64. Have not tried other architectures nor cross-compiling.

### CODE REVIEW

File: srcpkgs/hyperscan/template [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+	pkg_install() {
+		vmove usr/include
+		vmove "usr/lib/*.a"
+		vmove "usr/lib/*.so"
+		vmove usr/lib/pkgconfig
+	}
+}
+
+hyperscan-doc_package() {
+	short_desc+=" - documentation files"
```
Comment (xtraeme) [inline srcpkgs/hyperscan/template]: archs=noarch

Comment (non-Jedi) [reply srcpkgs/hyperscan/template]: done

**Comment by non-Jedi** [comment]
Seems like CI cross-compiling failed because cmake couldn't find python. Is a python interpreter available for some reason when not cross compiling but unavailable otherwise?

**Comment by Chocimier** [comment]
python is probably pulled as boost's makedepend dependency. As it is needed, put it into hostmakedepends. Also please remove slashes from symlinks.

**Comment by non-Jedi** [comment]
Ah, right. Makes sense. I've force-pushed with symbolic links fixed and python added to hostmakedepends.

**Comment by non-Jedi** [comment]
Force pushed with `archs=` set since intel doesn't support non-intel architectures. Also trying to figure out the build failure on musl i686 and x86_64. I'm trying to get xbps-src setup locally to be able to test compiling on musl, but I'm not there yet.

**Comment by non-Jedi** [comment]
Need to figure out why hyperscan SSSE3 testing fails with musl but not glibc. Looks like the following program fails to compile:

```C
#include <${INTRIN_INC_H}>
int main() {
    __m128i a = _mm_set1_epi8(1);
    (void)_mm_shuffle_epi8(a, a);
}
```

Is it okay if I just mark musl as broken until then?

**Pull request closed without merging**